commerce_coordinator/apps/frontend_app_payment/views.py

```diff
@@ -57,3 +62,20 @@ def put(self, request):
         params = input_serializer.data
         payment_details = DraftPaymentRequested.run_filter(params)
         return Response(payment_details)
+
+
+class GetActiveOrderView(APIView):
+    """Get Active Order View"""
+    authentication_classes = (JwtAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request):  # pylint: disable=inconsistent-return-statements
+        """return the user's current active order"""
+        params = {
+            'edx_lms_user_id': request.user.lms_user_id
+        }
+        input_serializer = GetActiveOrderInputSerializer(data=params)
+        if input_serializer.is_valid(raise_exception=True):
```

Comment:
as we have passed `raise_exception=True` in params, We can remove this `if` condition before.
it will become `input_serializer.is_valid(raise_exception=True)`.
This way you would be able to remove `# pylint: disable=inconsistent-return-statements`

Reply:
removed if statement